nss-day-cohort-47/reactive-nutshell-palm-beach-pilots
# Friends final

# Description
This adds the functionality of the friends component. It allows users to add and remove Friends from there profile.
Fixes # (issue)
## Type of change
Please delete options that are not relevant.
# Testing Instructions
User should click on the friends tab to see a list of friends or to add a new friend from the list of users. To delete a user the user should click on remove Friend Button. To add a user they should start typing the user name until the correct user is suggested. then tab and press add friend.
# Checklist:
- [ x] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [x ] My changes generate no new warnings or errors
- [x ] I have added test instructions that prove my fix is effective or that my feature works
